src/components/DpDetails/DpDetails.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpDetails from './DpDetails.vue'
+
+const meta: Meta<typeof DpDetails> = {
+    component: DpDetails,
+    title: "Components/Details",
+    argTypes: {},
+    render: (args) => ({
+        components: {
+            DpDetails,
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-details primary secondary v-bind="args">
```

Comment:
I think `primary` and `secondary` should be removed here

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/400/commits/9ececfdfdb0bb23ebedd09097c8aac43c61d79d6